esmvaltool/cmorizers/data/formatters/datasets/mls_aura.py

```diff
@@ -19,7 +19,6 @@
     control of the RHI data (see Data Quality Document for MLS-AURA for more
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Fixed